website/docs/advanced/verification-methods.md

```diff
@@ -4,16 +4,25 @@ title: Extending Verification Methods
 sidebar_label: Verification Methods
 ---
 
-As explained by our [Verifier ADR](https://github.com/Open-Attestation/adr/blob/master/verifier.md), the library is composed of verification methods, that are run over a document and provide fragments containing details about each verification. In this guide we will learn how to write custom verification method and how you can distribute your own verifier.
+As explained by our [Verifier ADR](https://github.com/Open-Attestation/adr/blob/master/verifier.md), the library is composed of verification methods that are run over a document and provide fragments containing details about each verification. In this guide, we will use the contents in a _Version 2_ document as an example and learn how to write custom verification methods and distribute your own custom verifier.
```

Comment:
Our doc is not clear on version 2 / 3. It will only confuse people. Let's revert the part about version 2. Can keep the rest

Reply:
ok done.